angular-workspace/projects/ni/nimble-angular/src/directives/table/tests/nimble-table.directive.spec.ts

```diff
@@ -137,6 +137,7 @@ describe('Nimble table', () => {
         it('`checkValidity()` returns `false` when the table is not valid', () => {
             fixture.componentInstance.idFieldName = 'not-a-field';
             fixture.detectChanges();
+            processUpdates();
```

Comment:
Unfortunately, I think the answer to "how does this impact the way client apps will write their tests" is "it depends". With this change, I didn't have to update all the tests within nimble-components because they already had `await waitForUpdatesAsync()` calls in them because of other timing considerations. The one change I had to make to a nimble-components test was to call `waitForUpdatesAsync()` after `await connect()`, but that call was already present in most existing tests.

At this point in time, I'm not sure what requirement I'd want to capture, and putting something in the page object doesn't make sense because right now our page object is more about querying the table than manipulating it (which is where the waits need to happen). My current thought is that we do nothing for now other than keep it in mind when we get to the point of thinking about how clients would write tests that interact with the table.

Reply:
I'm fine with that, thanks for thinking about it.